src/pages/settings/Payments/AddDebitCardPage.js

```diff
@@ -120,6 +120,8 @@ class DebitCardPage extends Component {
                     validate={this.validate}
                     onSubmit={PaymentMethods.addPaymentCard}
                     submitButtonText={this.props.translate('common.save')}
+                    scrollContextEnabled
+                    scrollToOverflowEnabled
```

Comment:
I think that we have to add these props to a few more pages, e.g. [CompanyStep](https://github.com/Expensify/App/blob/7ba22bdc256ad16f517102d8a12db7137c7a150a/src/pages/ReimbursementAccount/CompanyStep.js), [AdditionalDetailsStep](https://github.com/Expensify/App/blob/7ba22bdc256ad16f517102d8a12db7137c7a150a/src/pages/EnablePayments/AdditionalDetailsStep.js), otherwise we have the issue with the Picker being hidden in iOS:

https://user-images.githubusercontent.com/22219519/213524511-8a01f6ed-3d9f-4cbc-b9b2-bcba35f784f6.mov

Reply:
Done! I added it to all remaining pages/forms that contains (a) picker(s). In [27f5fea](https://github.com/Expensify/App/pull/14409/commits/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140), i also added it to some pages/forms, that seem like they are currently not in use, such as [BankAccountPlaidStep.js](https://github.com/margelo/expensify-app-fork/blob/27f5fea1c3562a8cd30b9a2b5ecf4373f39e0140/src/pages/ReimbursementAccount/BankAccountPlaidStep.js)